packages/console/src/components/Executions/Tables/WorkflowExecutionTable/styles.ts

```diff
@@ -10,6 +10,10 @@ export const useStyles = makeStyles((theme: Theme) => ({
     flexBasis: workflowExecutionsTableColumnWidths.name,
     whiteSpace: 'normal',
   },
+  columnLaunchPlan: {
```

Comment:
Wondering how this would handle a really large string; could we add something like `overflow:hidden` here?